Lib/test/mp_preload_flush.py

```diff
@@ -0,0 +1,14 @@
+import multiprocessing
+import sys
+
+modname = 'preloaded_module'
+if __name__ == '__main__':
+    assert modname not in sys.modules
```

Comment:
Please avoid assert keyword, but replace it with a test + raise on error, as done below.